src/components/Main.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
```

Comment:
Remove this import from react. Since you're using the new version of react. We no longer need it.

Reply:
Yeah sure!
Thanks